integration-tests/validation.py

```diff
@@ -255,13 +273,17 @@ def validate_status_messages(model):
     return True
 
 
+# note that name_list is a list of entities(pods, services, etc) being searched
+# and that partial matches work. If you have a pod with random characters at the
+# end due to being in a deploymnet, you can add just the name of the deployment
+# and it will still match
```

Comment:
Not nak-worthy, but consider jamming this wall of text into a docstring for this function.